src/main/java/appeng/tile/inventory/IAEItemFilter.java

```diff
@@ -0,0 +1,10 @@
+package appeng.tile.inventory;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+public interface IAEItemFilter 
+{
+	boolean allowExtract(IItemHandlerModifiable inv, int slot, int amount );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)